chart-templates/crd-base/overlay-upstream/templates/validate-install-crd.yaml

```diff
@@ -0,0 +1,12 @@
+# {{- \$found := dict -}}
+${set_found_crd}
```

Comment:
What is this used for?

Reply:
This comes directly from our original scripts at https://github.com/rancher/charts/blob/153171167ff52f0311b9a4d9968d33742d248101/scripts/prepare-crds#L97-L112, but the purpose of the `validate-install-crd.yaml` is to cause the original chart to fail on rendering (i.e. `helm install --debug --dry-run`) if the CRDs are not currently installed onto the cluster.

Some of the caveats with this approach are detailed in https://github.com/rancher/charts/pull/595#discussion_r483970192.

Reply:
but do you still these commented lines? if not I think we should remove it

Reply:
We need the commented lines since Helm doesn't know how to deal with an empty file that is generated if the CRDs are installed onto the cluster (i.e. since the file is empty, Helm will complain it is invalid YAML). However, a file which renders comments doesn't have this issue.